docs/references/go/overview.mdx

```diff
@@ -19,14 +19,14 @@ $ go get github.com/clerkinc/clerk-sdk-go
 
 ## `Clerk` client
 
-Now, you can create a `Clerk` client by calling the `clerk.NewClient` function. This function requires your Clerk secret key. If you are signed into your Clerk Dashboard, your keys should become visible by clicking on the eye icon. You can also retrieve your Clerk secret key from the Clerk Dashboard on the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys)** page.
+Now, you can create a `Clerk` client by calling the `clerk.NewClient` function. This function requires your Clerk secret key. If you are signed into your Clerk Dashboard, your secret key should become visible by clicking on the eye icon. Otherwise, you can retrieve your Clerk secret key from the Clerk Dashboard on the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys)** page.
```

Comment:
```suggestion
Now, you can create a `Clerk` client by calling the `clerk.NewClient()` function. This function requires your Clerk secret key. If you are signed into your Clerk Dashboard, your secret key should become visible by clicking on the eye icon. Otherwise, you can retrieve your Clerk secret key from the Clerk Dashboard on the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys)** page.
```